api/src/main/java/org/openmrs/module/sync2/client/rest/impl/RestResourceConverterImpl.java

```diff
@@ -150,4 +155,8 @@ private void convertForm(SimpleObject simpleObject) {
 		simpleObject.remove("resources");
 		simpleObject.remove("template");
 	}
+
+	private void convertCohort(SimpleObject simpleObject) {
```

Comment:
```suggestion
	private void convertCohort(Map<String, Object> simpleObject) {
```
Please use Map<String, Object>  since it is compatible with SimpleObject. 
Consider this line https://github.com/openmrs/openmrs-module-sync2/blob/8a8d83dff813814a69d4f433310c632234264519/api/src/main/java/org/openmrs/module/sync2/client/rest/impl/RestResourceConverterImpl.java#L119
When we call simpleObject.get("person") the returned value don't have SimpleObject type (probably it will be LinkedHashMap after parsing), so when we will pass this to convertPersonResource method we will get casting exception